e2e/atlas/logs_test.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e,atlas,logs
+
+package atlas_test
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/mongodb/mongocli/e2e/atlas"
+)
+
+func TestAtlasLogs(t *testing.T) {
+	cliPath, err := filepath.Abs("../../bin/mongocli")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	_, err = os.Stat(cliPath)
+
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	atlasEntity := "atlas"
+	logsEntity := "logs"
+	clusterName := ""
+
+	// Deploy a cluster only if there is not one available
+	if !atlas.ExistCluster(cliPath) {
```

Comment:
if you plan to run on parallel you need each test to create their own cluster or that cluster can be deleted in the middle of the test by the other test that created it